Mobile/Android/Templates/Release_2.5.0/xxxx-oauth2.md

```diff
@@ -0,0 +1,54 @@
+###  OAuth2 
+
+#### Pr: https://github.com/owncloud/android/pull/XXX 
+
+
+---
+
+ 
+| TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+| :----: | :-------- | :---- | :-------------- | :----: | :-------------- |
+|**Login View**|||||||
+| 1 | Initial look | Open the app. Check both orientations | Correct view. Only URL and connect button are shown  |  |  |
+| 2 | OAuth2 URL | 1. Set an URL of a server with OAuth2<br>2. Tap on connect.<br> Check both orientations | Redirected to webview. No more options shown |  |  |
+| 3 | Basic Auth URL | 1. Set an URL of a server without OAuth2<br>2. Tap on connect.<br>Check both orientations | Fields username and password are shown |  |  |
+| 4 | Redirected URL | 1. Set a redirection URL <br>2. Tap on connect. <br>Check both orientations | Fields username and password are shown  |  |
+| 5 | SAML URL | Set an SAML URL (not supported by the app)<br>Check both orientations | Correct error messaege |  |  |
+| 6 | Wrong URL | Set an incorrect URL. <br>Check both orientations | Correct error messaege |  |  |
+| 7 | Wrong Authorizathion endpoint | 1. In customization file, change auth endpoint for a incorrect one<br>2. Enter OAuth2 URL and connect. <br>Check both orientations | Correct error handling |  |  |
```

Comment:
i know it. Maybe some options will not be brandeable for this approach, but in any case can be set up using customization files.